Core/Core.cpp

```diff
@@ -208,57 +196,40 @@ bool UpdateScreenScale(int width, int height) {
 	return false;
 }
 
-// Note: not used on Android.
-void UpdateRunLoop() {
+// Used by Windows, SDL, Qt.
+void UpdateRunLoop(GraphicsContext *ctx) {
+	NativeFrame(ctx);
 	if (windowHidden && g_Config.bPauseWhenMinimized) {
 		sleep_ms(16);
 		return;
 	}
-	NativeFrame(graphicsContext);
-}
-
-void KeepScreenAwake() {
-#if defined(_WIN32) && !PPSSPP_PLATFORM(UWP)
-	SetThreadExecutionState(ES_SYSTEM_REQUIRED | ES_DISPLAY_REQUIRED);
-#endif
 }
 
+// Note: not used on Android.
 void Core_RunLoop(GraphicsContext *ctx) {
 	float refreshRate = System_GetPropertyFloat(SYSPROP_DISPLAY_REFRESH_RATE);
 
-	graphicsContext = ctx;
-	while ((GetUIState() != UISTATE_INGAME || !PSP_IsInited()) && GetUIState() != UISTATE_EXIT) {
-		// In case it was pending, we're not in game anymore.  We won't get to Core_Run().
-		Core_StateProcessed();
-		double startTime = time_now_d();
-		UpdateRunLoop();
+	if (windowHidden && g_Config.bPauseWhenMinimized) {
+		sleep_ms(16);
+		return;
+	}
+
+	bool menuThrottle = (GetUIState() != UISTATE_INGAME || !PSP_IsInited()) && GetUIState() != UISTATE_EXIT;
 
+	// In case it was pending, we're not in game anymore.
```

Comment:
If anything this is probably more related to `menuThrottle`?  I think this was about the call to Core_StateProcessed anyway.

-[Unknown]